### PR TITLE
feat: make brand search look like a brand page

### PR DESCRIPTION
**Pull request by mzanoni**
https://trello.com/c/fiHBNyXV/8698-demo-shop-make-brand-search-look-like-a-brand-page

### CODE REVIEW

File: src/components/Facets.vue [hunk 1]
```diff
@@ -81,6 +81,7 @@ const props = defineProps({
     facets: { type: Object as PropType<ProductFacetResult>, required: true },
     page: { type: Number, required: true },
     renderCategoryFacet: { type: Boolean, required: true },
+    hideBrandFacet: { type: Boolean, default: false },
```
Comment (SWH-Relewise) [inline src/components/Facets.vue:84]: Skal vi ikke aligne de her 2? 😊

**Review by SWH-Relewise — APPROVED** [review]
Awesome! 🥳

File: src/components/Facets.vue (hunk 1 above)
Comment (mzanoni) [reply src/components/Facets.vue:84]: Men jeg vil meget gerne defaulte til at vise brand facetten :(

Comment (SWH-Relewise) [reply src/components/Facets.vue:84]: Jamen så kan vi jo bare lave den anden om 😊

**Review by SWH-Relewise — APPROVED** [review]

**Pull request merged**